### ISSUE TITLE
Issue during packaging 

### ISSUE THREAD

**Issue by Vanjiga** [wsteenhuis/MarkovBot#1]
From what I see dependencies have to be updated. 
>mvn package
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------< wsteenhuis:MarkovBot >------------------------
[INFO] Building MarkovBot 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.611 s
[INFO] Finished at: 2021-06-23T01:18:37+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project MarkovBot: Could not resolve dependencies for project wsteenhuis:MarkovBot:jar:1.0-SNAPSHOT: Failed to collect dependencies at com.github.austinv11:Discord4j:
jar:shaded:2.9.2 -> jflac:jflac:jar:1.3: Failed to read artifact descriptor for jflac:jflac:jar:1.3: Could not transfer artifact jflac:jflac:pom:1.3 from/to maven-default-http-blocker (http://0.0.0.0/
): Blocked mirror for repositories: [jcenter (http://jcenter.bintray.com, default, releases+snapshots)] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

Thank you for your time to read this and hopefully a fast fix

**Comment by ghost** [comment]
Having the same issue... Figure it's been too long and this bot is obsolete.

**Comment by sub-res** [comment]
Yup, this bot is pretty much ancient by this point and to be honest it would require a complete rewrite by this point anyway. Not sure what the best course of action here is, I haven't touched this project in years now and the spaghetti nature of its code kind of makes me not want to either.

Maybe it's best that I private this repo eventually because of its myriad issues, apologies for the inconvenience.
